timeless/views.py

```diff
@@ -57,7 +56,8 @@ class CrudAPIView(views.MethodView):
 
     def get(self):
         """Calls the GET method."""
-        pass
+        model_id = request.form.get(self.url_lookup)
+        return self.model.query.get(int(model_id))
```

Comment:
@salgadobreno you return model instance on GET request. it's definitely incorrect behavior. You should return serialized presentation of object. I think we should use something like this: https://marshmallow-sqlalchemy.readthedocs.io/en/latest/

Reply:
@vryazanov Query is defined as an abstraction to decouple DB objects as mentioned in puzzle at line 5.. From FakeQuery(defined in the same file) and the test it seemed it should be returning a dict.. I think you're right but also the design and documentation may need improvement. I'll make FakeQuery return a jsonified object, though. Thanks!

Reply:
@salgadobreno I do not know what is `FakeQuery` (I think it's rudiment) and why we have it in the code. But current implementation of `get` query does not make any sense, since it should return `Response` object (or `dict`, not sure), but not an instance of object. I think we can leave a puzzle to bring https://marshmallow-sqlalchemy.readthedocs.io/en/latest/ or something like that to serialize objects.

Reply:
@salgadobreno leave a puzzle for this as well